docs/en/operation/User-CLI.md

```diff
@@ -229,12 +229,13 @@ Status: CANCELED
 The `logLevel` command returns the current value of or updates the log level of a particular class
 on specific instances. Users are able to change Alluxio server-side log levels at runtime.
 
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:port>] [--level=LEVEL]`,
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:webPort[:role]>] [--level=LEVEL]`,
 where:
 * `--logName <arg>` indicates the logger's class (e.g. `alluxio.master.file.DefaultFileSystemMaster`)
 * `--target <arg>` lists the Alluxio master or workers to set.
-The target could be of the form `<master|workers|job_master|job_workers|host:webPort>` and multiple targets can be listed as comma-separated entries.
-The `host:webPort` format can only be used when referencing a worker.
+The target could be of the form `<master|workers|job_master|job_workers|host:webPort[:role]>` and multiple targets can be listed as comma-separated entries.
+`role` can be one of `master|worker|job_master|job_worker`. Using the `role` option is useful when an Alluxio process
```

Comment:
before, i could call `--target=master` or `--target=worker:workerWebPort`. with the new role addition, i'm not sure what the format is with when trying to use `role` but instinctively it looks like `targetType[:portIfWorker][:role]. is this accurate? maybe worth giving some examples as opposed to adding more syntax to the existing command

Reply:
+1 on providing examples on when/how `:role` is used

Reply:
I added more examples.